.github/workflows/cicd-pipeline.yml

```diff
@@ -60,6 +78,20 @@ jobs:
         name: webapp-test-report.log
         path: webapp-test-report.log
 
+    - name: Archive test screenshots
+      uses: actions/upload-artifact@v1
+      with:
+        name: webapp-e2e-screenshots
+        path: webapp/client/cypress/screenshots
+      if: ${{ failure() }}
+
+    - name: Archive test videos
```

Comment:
Hm, not sure? I actually like the symmetry to the acceptance tests job…

Reply:
Okay, it just got me confused because I was wondering wether this replaces the acceptance tests. However, this might be more a problem of reading a PR then reading the CICD pipeline itself.

Reply:
Probably also due to the confusion in how I named this test suite and what differentiates it from acceptance tests…